gulpfile.js

```diff
@@ -72,20 +60,29 @@ gulp.task('jsBuild', function() {
 });
 
 gulp.task('imgMove', function() {
-  return gulp.src('public/images/**/*')
-    .pipe(cache('imgMove'))
-    .pipe(gulp.dest('build/public/images'));
+    return gulp.src('public/images/**/*')
+        .pipe(cache('imgMove'))
+        .pipe(gulp.dest('build/public/images'));
 });
 
 gulp.task('clean', function(callback) {
     cache.caches = {};
-    return del(['build/**/*', '!node_modules/**', '!bower_components/**'], callback);
+    return del(['build/**/*'], callback);
 });
 
 gulp.task('build', function(callback) {
     return runSequence('clean', ['jsBuild', 'sassBuild', 'imgMove', 'moveViews'], callback);
 });
 
+gulp.task('productionBuild', function(callback) {
+    return runSequence('build', 'moveDependencies', callback);
```

Comment:
we need to work with this in the react branch, which has browserify. Ideally, `productionBuild` will basically just be a `--bare` browserify on `www.js` that smushes all the node code / deps into one file.